scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryClient.scala

```diff
@@ -449,6 +455,14 @@ class BigQueryClient private (private val projectId: String,
     }
   }
 
+  private def getUdfResource(): Option[java.util.List[UserDefinedFunctionResource]] = {
+    BigQueryClient.udfResourceUri.map { uri =>
```

Comment:
How do you pass in the resource URI for standard SQL then? As part of the query string?